src/array_util.rb

```diff
@@ -0,0 +1,43 @@
+class ArrayUtil
+
+  def self.sum_even_numbers(numbers)
+    numbers&.select(&:even?)&.sum
+  end
+
+  def self.two_largest_numbers(numbers)
+    return nil if numbers.nil? ||  numbers.size < 2
+
+    firstNumber = numbers[0]
+    secondNumber = numbers[1]
+
+    if firstNumber < secondNumber
+      firstNumber, secondNumber = secondNumber, firstNumber
+    end
+
+    numbers[2...numbers.size].each do |x|
+      if x > firstNumber
+        secondNumber = firstNumber
```

Comment:
Why did you choose to do the assignments separately here and initially you condensed them to one line?

What is the better approach in your opinion?

Reply:
I was reading about Ruby conventions: https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md
But I miss something: <i>"Prefer {...} over do...end for single-line blocks. Avoid using {...} for multi-line blocks (multiline chaining is always ugly)"</i>
So, this approach is better for multi-line.

Reply:
This is not what I meant, I was referring to:

```ruby
 +      firstNumber, secondNumber = secondNumber, firstNumber 
```

and

```ruby
secondNumber = firstNumber
firstNumber = x
```

Reply:
Because if I have ```numbers = [1,2,3]``` the final result will be ```[3,1]``` instead of ```[3,2]``` without initial switch.
So, I check that the largest value is set to ```firstNumber```.